container/Dockerfile_local

```diff
@@ -11,9 +13,9 @@ WORKDIR '/faf'
 # Change owner of /faf, clean git and install dependences
 RUN chown -R --silent faf /faf && \
     chmod -R --silent g=u /faf && \
-    dnf -y install rpm-build sudo git which vim make &&  \
+    dnf -y install git-core make rpm-build sudo tito vim which && \
```

Comment:
Do you think it would be worth it to move this installation above `COPY . /faf` so that it leverage the cache better?

Reply:
Ok, I did something. 

Please review.